data/data/azure/bootstrap/main.tf

```diff
@@ -216,6 +216,8 @@ resource "azurerm_linux_virtual_machine" "bootstrap" {
 }
 
 resource "azurerm_network_security_rule" "bootstrap_ssh_in" {
+  count = var.private ? 0 : 1
```

Comment:
I thought that we had talked about keeping this security rule for private clusters but changing the allowed source addresses to the vnet cidrs, but I may be forgetting the details. Did we decide that was not needed because, with Azure, the intra-vnet traffic is all allowed?

Reply:
That is my understanding. When you create a Network Security Group in azure, it will allow intra VNET traffic ([default NSG security rule](https://docs.microsoft.com/en-us/azure/virtual-network/network-security-groups-overview#default-security-rules))